src/ape_plugins/_cli.py

```diff
@@ -181,8 +213,17 @@ def install(cli_ctx, skip_confirmation):
             any_install_failed = True
         # check for installed check the config.yaml
         elif installed_plugin and available_plugin:
-            cli_ctx.logger.warning(f"Plugin '{module_name}' is trusted and already installed.")
-            pass
+            cli_ctx.logger.info(f"Plugin '{module_name}' is trusted and already installed.")
```

Comment:
Same thing here - we should move this to the other side of the `if upgrade` in an `else` block so it only happens when the user forgot to include `--upgrade` and the package is already installed. We should also warn them to use `--upgrade` if they wish to upgrade, in case they mistakenly forgot `--uprade`